tRecorderApi/api/views/version_list.py

```diff
@@ -11,6 +11,7 @@ class getVersionsView(views.APIView):
     def post(self, request):
         data = json.loads(request.body)
         allTakes = Take.objects.all().values()
+        allTakes = allTakes.filter(is_source = False)
```

Comment:
@lcheng49 Can we init and assign the variable in one line? Does it need to be 2?